.azure-pipelines/powershell-core.yml

```diff
@@ -18,9 +18,6 @@ jobs:
       windows:
         OSName: ${{ variables.WindowsName }}
         ImageName: ${{ variables.WindowsImage }}
-      linux:
```

Comment:
@isra-fel It seems master has temporarily patched the linux build pipeline failure, so I also copied the content here to fix the build error. After this PR is merged I think we need another PR to sync master and Az.Resources-preview. Should it be done by us or the powershell team?

Reply:
It will be just like normal PRs - you submit it, we review it.